flink-java/src/test/java/org/apache/flink/api/common/operators/CollectionExecutionAccumulatorsTest.java

```diff
@@ -35,7 +35,7 @@
 	@Test
 	public void testAccumulator() {
 		try {
-			final int NUM_ELEMENTS = 100;
+			final Integer NUM_ELEMENTS = 100;
```

Comment:
Can we stay, whereever possible, with primitive types? Rather unbox the type at a different place?